app/components/primer/dropdown_component.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Primer
+  # Dropdowns are lightweight context menus for housing navigation and actions.
```

Comment:
I wonder if we could clarify when to use `Dropdown` and when to use `DropdownMenu`. It's not clear to me if we ever need a `DropdownMenu` now that we have this component...

Reply:
I see `DropdownMenu` as a subcomponent of `Dropdown`, it is better to have them separate since it simplifies the logic in each one

Reply:
I agree with @srt32. Are there any cases in GitHub.com where we'd need both components?

Reply:
They always go together, but I'm keeping them in separate files so we don't end up with a 300+ lines component.
As of why we need both is because it keeps the API cleaner, since I can customize the `Menu` with direction, alignment, paddings...

One thing I can do is change `Primer::DropdownMenuComponent` to `Primer::Dropdown::MenuComponent` like I did with SelectMenu to make it clearer that this is a subcomponent that shouldn't be used alone